src/repository/storage/postgres/orm/sequelize/teams.ts

```diff
@@ -220,4 +220,24 @@ export default class TeamsSequelizeStorage {
 
     return affectedRows > 0;
   }
+
+  /**
+   *
+   * @param userId - id of team member
+   * @param noteId - note internal id
+   * @param role - new team member role
+   */
+  public async patchMemberRoleById(userId: TeamMember['id'], noteId: NoteInternalId, role: keyof typeof MemberRole): Promise<keyof typeof MemberRole | null> {
+    const affectedRows = await this.model.update({
+      role: MemberRole[role],
```

Comment:
why not just `role`?

Reply:
> role is < "write" | "read"

why? I see enum with 0 and 1

Reply:
now we use MemberRoleKeys (special type `'read' | 'write'`) and MemberRole[role] is number

Reply:
changed